dev/integration_tests/android_views/README.md

```diff
@@ -0,0 +1,32 @@
+# Integration test for touch events on embedded Android views
```

Comment:
Are you adding more generic Android view embedding tests to this or is it just for touch events? (just wondering if you're going to want to use this generic name later)